resources/pool.rb

```diff
@@ -80,7 +80,7 @@
 property :smp_processor_affinity_mask, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 property :smp_processor_affinity_mask_2, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 
-property :running, [true, false], desired_state: true
+property :running, [true, false]
```

Comment:
Why remove the `desired_state: true` from the `running` property?  It's not a settable property by the user right?